apps/examples/testcase/ta_tc/mqtt/itc/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_MQTT_ITC
+	bool "MQTT ITC TestCase Example"
+	depends on NETUTILS_MQTT
+	default n
+	---help---
+		Enable the MQTT TestCase example
+
+if EXAMPLES_TESTCASE_MQTT_ITC
+
+config EXAMPLES_TESTCASE_MQTT_BROKER_ADDR
```

Comment:
EXAMPLES_TESTCASE_MQTT_BROKER_ADDR and EXAMPLES_TESTCASE_MQTT_BROKER_PORT are same with utc's.
It causes re-definitions.

Reply:
Currently we have added only basic scenarios. We will be adding more test scenarios with using MQTT APIs.
Also we will change parameter names according to ITCs to avoid re-definitions.

Reply:
@arvinmittal For dm, iotbus and arastorage, I heard same opinion a year ago, but there is no activity until now. Could you share a plan for adding additional scenarios?